var/spack/repos/builtin/packages/py-cinemasci/package.py

```diff
@@ -11,12 +11,13 @@ class PyCinemasci(PythonPackage):
     databases"""
 
     homepage = "https://github.com/cinemascience"
-    pypi = "cinemasci/cinemasci-1.3.tar.gz"
+    pypi = "cinemasci/cinemasci-1.7.0.tar.gz"
 
     tags = ['e4s']
 
     maintainers = ['EthanS94']
 
+    version('1.7.0', sha256='70e1fa494bcbefdbd9e8859cdf1b01163a94ecffcdfa3da1011e4ef2fcee6169', preferred=True)
```

Comment:
```suggestion
    version('1.7.0', sha256='70e1fa494bcbefdbd9e8859cdf1b01163a94ecffcdfa3da1011e4ef2fcee6169')
```
Spack always prefers the latest stable release by default, no need to specify this